### PR TITLE
Use new images

### PR DESCRIPTION
**Pull request by pameladelgado**

### CODE REVIEW

**Comment by rokroskar** [comment]
also bump the renku-python version to 0.14.1 otherwise new projects are immediately asked to be migrated

File: bioc-minimal/Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # see https://github.com/SwissDataScienceCenter/renkulab-docker
 # to swap this image for the latest version available
-FROM renku/renkulab-bioc:RELEASE_3_11-0.7.3
+FROM renku/renkulab-bioc:RELEASE_3_11-0.7.5
```
Comment (gavin-k-lee) [inline bioc-minimal/Dockerfile:3]: Could we use 3.12? :) Users have been asking for this for a bit.

**Comment by rokroskar** [comment]
We should also update the python version in the manifest so it says "Python 3.8" not 3.7

**Comment by pameladelgado** [comment]
> We should also update the python version in the manifest so it says "Python 3.8" not 3.7

good point, done.

**Review by gavin-k-lee — APPROVED** [review]

**Pull request merged**